src/logger/src/logger.rs

```diff
@@ -645,10 +692,38 @@ mod tests {
         logger.mock_log(Level::Warn, "msg");
         validate_log(
             &mut Box::new(&mut reader),
-            "[TEST-INSTANCE-ID:WARN:logger.rs:0] msg\n",
+            &format!(
+                "[TEST-INSTANCE-ID:{}:WARN:logger.rs:0] msg\n",
+                crnt_thread_name
+            ),
         );
     }
 
+    #[test]
+    fn test_thread_name_custom() {
+        // Name this thread "custom-thread"
```

Comment:
This comment can be removed, since the name of the test is pretty self-explanatory.